legend-shared-pac4j-gitlab/src/main/java/org/finos/legend/server/pac4j/gitlab/GitlabTokenClient.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.server.pac4j.gitlab;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.finos.legend.server.pac4j.SerializableProfile;
+import org.pac4j.core.client.DirectClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SerializableProfile
+public class GitlabTokenClient extends DirectClient<GitlabAuthUserCredentials, GitlabUserProfile>
```

Comment:
This should be renamed to make clear that it is an authenticator for personal access tokens.